electron/src/electron/modals.ts

```diff
@@ -0,0 +1,143 @@
+import { WebPlugin } from "@capacitor/core";
+
+declare var require: any
+const {dialog , getCurrentWindow} = require('electron').remote
+
+import {
+  ModalsPlugin,
+  AlertOptions,
+  PromptOptions,
+  PromptResult,
+  ConfirmOptions,
+  ConfirmResult,
+  ActionSheetOptions,
+  ActionSheetResult
+} from "@capacitor/core";
+
+export class ModalsPluginElectron extends WebPlugin implements ModalsPlugin {
+  resolve: any;
+  constructor() {
+    super({
+      name: 'Modals',
+      platforms: ['electron']
+    });
+  }
+  async alert(options: AlertOptions): Promise<void> {
+    //window.alert(options.message); we can't set title and button title
+    const alert = (message: string, title: string = '') =>
+    {    
+        
+        let buttons = [options.buttonTitle || 'OK']
+        dialog.showMessageBox(getCurrentWindow(), {message, title, buttons});       
+    }
+        alert(options.message, options.title);
+        return Promise.resolve();
+  }
+  //window.prompt is and will not be supported in Electron. Not working with Ionic components?
```

Comment:
First part of the comment might be good to know, but the second part, not sure what do you want to say with Not working with Ionic components?. Better not leave behind this kind of comments.